manifests/init.pp

```diff
@@ -195,6 +195,8 @@
 #   Close the connection after a client is idle for N seconds (0 to disable).
 # @param ulimit
 #   Limit the use of system-wide resources.
+# @param ulimit_managed
+#   Defines wheter the Limit is managed by this module or not.
```

Comment:
Maybe I'm to blame for this, but perhaps it's a good chance to clear up the text? As a user this doesn't really tell me what it does.